furthermaths/vectors/angle.py

```diff
@@ -1,10 +1,19 @@
 """Angle between two vectors"""
+from math import acos,sqrt
+from Vectors import Vector
+pi = 3.14159265359
 
-
-# TODO add angle between two vectors
 def angle(v1, v2):
-    pass
-
+    if not isinstance(v1,Vector) or not isinstance(v2,Vector):
+        return  ValueError("not a vector")
+    if v1.dimension!=v2.dimension:
+        raise ValueError("Dimensions of vectors should be equal")
+    dotProduct = sum(i * j for i, j in zip(v1.values, v2.values))
+    modOfVectors = sqrt(sum(i*i for i in v1.values)) * sqrt(sum(i*i for i in v2.values))
```

Comment:
```suggestion
    modOfVectors = (sum(i*i for i in v1.values) * sum(i*i for i in v2.values))**0.5
```
the point of this package is to accomplish this with as little as possible external packages